packcheck-safe.sh

```diff
@@ -1,9 +1,28 @@
 #!/usr/bin/env bash
 
-PACKCHECK_DIR=$(dirname $0)
+PACKCHECK_DIR=$(dirname "$0")
 echo "Running ${PACKCHECK_DIR}/packcheck.sh with clean environment and CHECK_ENV on..."
 echo "No environment variables are honored, you have to specifiy ALL the"
 echo "parameters explicitly on the command line, including PATH."
 echo
 
-/usr/bin/env -i CHECK_ENV=y $(dirname $0)/packcheck.sh $*
+PACKCHECK_CLI_OPTS_ARR=("$@")
```

Comment:
Just quoting `$*` should work here e.g. `/usr/bin/env -i CHECK_ENV=y $(dirname $0)/packcheck.sh "$*"`

Reply:
This does not work. `a="b c d"` would become `a=b c d`.